package.json

```diff
@@ -116,10 +116,12 @@
     "build": "webpack",
     "build:prod": "webpack --config webpack.prod.js",
     "clean": "rimraf ./dist",
+    "clear-react": "npm uninstall react",
+    "reinstall-react": "npm install -D react",
```

Comment:
Created some scripts that now allow yarn/npm link to finction properly. Whenever you make a change and want to run it in the other website, just use `npm run package`. React will be uninstalled from the node_modules directory when it's finished so that hooks will work in the other website.